### PR TITLE
DOCSP-9030: feedback fixes for Fundamentals > Connection Guide

### PR DESCRIPTION
**Pull request by ccho-mongodb**
JIRA: https://jira.mongodb.org/browse/DOCSP-9030

Staging:  https://docs-mongodborg-staging.corp.mongodb.com/0bbce61/node/docsworker-xlarge/DOCPS-9030-connection-guide-fixes//fundamentals/connection

### CODE REVIEW

File: source/fundamentals/connection.txt [hunk 1]
```diff
@@ -61,13 +62,15 @@ A MongoDB replica set deployment is a group of connected instances that
 store the same set of data. This configuration of instances provides data
 redundancy and high data availability.
 
-To connect to a replica set deployment, specify the host and port numbers of
-each instance, separated by a comma, and the replica set name as the value of
-the ``replicaSet`` parameter.
+To connect to a replica set deployment, specify the hostname and port numbers
+of each instance, separated by a comma, and the replica set name as the value
+of the ``replicaSet`` parameter in the connection string. You must use the
+protocol ``mongodb`` instead of ``mongodb+srv`` when you specify multiple
```
Comment (mbroadst) [inline source/fundamentals/connection.txt]: A user does not decide to use `mongodb+srv` or `mongodb`, they are generally just "given" one. If they are using Atlas, then Atlas will default to `mongodb+srv` which uses an [SRV record](https://tools.ietf.org/html/rfc2782) to convey multiple addresses for the seed list.

Comment (ccho-mongodb) [reply source/fundamentals/connection.txt]: 👍 I'll omit these two sentences.

**Comment by ccho-mongodb** [comment]
@mbroadst Ready for re-review when you get the chance, thanks.

**Comment by mbroadst** [comment]
@ccho-mongodb remember to press the "re-request review" button by my name in the "Reviewers" list, it will make sure this PR gets back into my GitHub notifications window

**Pull request merged**